scripts/record_books.py

```
@@ -65,25 +72,278 @@
+                res = sample_ntp(s)
+                if res:
+                    offset_ms, rtt_ms = res
+                    self.latest_ntp_offset_ms = offset_ms
+                    ts = time.time()
+                    try:
+                        self.conn.execute(
+                            "INSERT INTO ntp_samples (ts, server, offset_ms, rtt_ms) VALUES (?,?,?,?)",
+                            (ts, s, offset_ms, rtt_ms))
+                        self.conn.commit()
+                    except Exception:
+                        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)